pkg/azureclients/fileclient/azure_fileclient.go

```diff
@@ -84,11 +84,11 @@ func (c *Client) WithSubscriptionID(subscriptionID string) Interface {
 }
 
 // CreateFileShare creates a file share
-func (c *Client) CreateFileShare(ctx context.Context, resourceGroupName, accountName string, shareOptions *ShareOptions) error {
+func (c *Client) CreateFileShare(ctx context.Context, resourceGroupName, accountName string, shareOptions *ShareOptions, expand string) (storage.FileShare, error) {
```

Comment:
add comments for `expand` parameter, e.g. what's the available values

Reply:
ok, it's added.